src/chttpd/src/chttpd.erl

```diff
@@ -1051,6 +1045,149 @@ error_info({Error, Reason, _Stack}) ->
 error_info(Error) ->
     maybe_handle_error(Error).
 
+
+erlfdb_error_info({erlfdb_error, ErrorCode}) ->
+    ErrorName = fdb_error_name(ErrorCode),
+    ErrorDesc =  erlfdb:get_error_string(ErrorCode),
+    {500, ErrorName, ErrorDesc}.
+
+
+fdb_error_name(0) ->
```

Comment:
Since we would keep these in sync manually, wonder if we should allow a catch-all clause here so if FDB adds a new error code and we forget to add it, at least we'll get the numeric code in the error instead of a clause error.

Reply:
I think it'd probably make more sense to move this to erlfdb.

Reply:
Good idea since these are not CouchDB specific. Maybe `erlfdb:get_error_name/1` or similar.

Reply:
https://github.com/apache/couchdb-erlfdb/pull/13